packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1762,11 +1799,7 @@ export namespace WindowedList {
     /**
      * Items list to be rendered
      */
-    itemsList: {
-      get?: (index: number) => T;
-      length: number;
-      changed: ISignal<any, IObservableList.IChangedArgs<any>>;
-    } | null;
+    itemsList: ISimpleObservableList | null;
```

Comment:
And the generic needs to be passed here:

```suggestion
    itemsList: ISimpleObservableList<T> | null;
```